BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/image_maker.sh

```diff
@@ -56,23 +57,23 @@ module unload python/3.7-2019.10
 source $PSIMDIR/set_env.sh
 #source $PSIMDIR/set_env.sh
 
-echo "starting PORP"
-ls -alrt
+echo "starting Physics of Results Plots"
+#ls -alrt
 
-echo "max index: $max_index"
-echo "exp: $exp"
+#echo "max index: $max_index"
+#echo "exp: $exp"
 
 python physicsOfResultsPUEO.py $WorkingDir/Run_Outputs/$RunName/Root_Files/${gen}_Root_Files $WorkingDir/Run_Outputs/$RunName/Generation_Data $max_index $exp
 
-mkdir ${destinationDir}/${gen}_detector_photos
-mv ${photoSourceDir}/${max_index}_detector.png ${destinationDir}/${gen}_detector_photos/${gen}_${max_index}_detector_max.png
-mv ${photoSourceDir}/${mid_index}_detector.png ${destinationDir}/${gen}_detector_photos/${gen}_${mid_index}_detector_mid.png
-mv ${photoSourceDir}/${min_index}_detector.png ${destinationDir}/${gen}_detector_photos/${gen}_${min_index}_detector_min.png
-#
-### Removes the remaining photos
-rm ${photoSourceDir}/*_detector.png
+echo "finished Physics of Results Plots"
+
+echo "copying best detector photos"
+
+mkdir -p ${destinationDir}/LabelledDetectorPhotos/${gen}_labelled_photos
+cp ${photoSourceDir}/${max_index}_detector.png ${destinationDir}/LabelledDetectorPhotos/${gen}_labelled_photos/${gen}_${max_index}_detector_max.png
+cp ${photoSourceDir}/${mid_index}_detector.png ${destinationDir}/LabelledDetectorPhotos/${gen}_labelled_photos/${gen}_${mid_index}_detector_mid.png
+cp ${photoSourceDir}/${min_index}_detector.png ${destinationDir}/LabelledDetectorPhotos/${gen}_labelled_photos/${gen}_${min_index}_detector_min.png
 #
-doubleNPOP=$((NPOP*2))
```

Comment:
Is this unimportant? I remember implementing this in a few places for the purpose of accounting for symmetry--should it be a switch here or is it unnecessary?

Reply:
I changed it to input the SYMMETRY variable into polar_plotter, and if symmetry is on it loops through npop, and if symmetry is off it loops through npop * 2